pkg/driver_atwinc15x0/patches/0001-change-CONST-to-_CONST_.patch

```diff
@@ -1,71 +0,0 @@
-From f07d66798c27359b4348e4a4842b55f601068bd0 Mon Sep 17 00:00:00 2001
-From: Gunar Schorcht <gunar@schorcht.net>
-Date: Thu, 26 Mar 2020 15:26:52 +0100
-Subject: [PATCH 1/6] change CONST to _CONST_
```

Comment:
This patch has to be kept, otherwise compilation fails:
```
In file included from drivers/include/net/netdev.h:202:0,
                 from drivers/include/atwinc15x0.h:25,
                 from drivers/atwinc15x0/include/atwinc15x0_internal.h:22,
                 from drivers/atwinc15x0/atwinc15x0_netdev.c:25:
core/lib/include/kernel_defines.h:94:0: error: "CONST" redefined [-Werror]
 #define CONST  __attribute__((const))
 
In file included from drivers/include/atwinc15x0.h:23:0,
                 from drivers/atwinc15x0/include/atwinc15x0_internal.h:22,
                 from drivers/atwinc15x0/atwinc15x0_netdev.c:25:
build/pkg/driver_atwinc15x0/src/bsp/include/nm_bsp.h:55:0: note: this is the location of the previous definition
 #define CONST const
 
In file included from drivers/atwinc15x0/atwinc15x0_netdev.c:30:0:
build/pkg/driver_atwinc15x0/src/driver/include/m2m_wifi.h:1488:1: error: 'const' attribute ignored [-Werror=attributes]
 NMI_API sint8 m2m_wifi_enable_ap(CONST tstrM2MAPConfig* pstrM2MAPConfig);
 ^~~~~~~
```

Reply:
Ah ok, I missed the dependency on PR #18212.